src/main/java/mate/academy/service/MovieService.java

```diff
@@ -9,4 +9,8 @@ public interface MovieService {
     Movie get(Long id);
 
     List<Movie> getAll();
+
+    Boolean update(Movie movie);
```

Comment:
Consider changing the return type of the `update` method to `Movie` or `void` to better reflect its operation. Returning a Boolean might not provide enough information about the update process.